projects/playground/src/app/app.component.ts

```diff
@@ -36,6 +38,12 @@ export class AppComponent {
     ignoreCache: new FormControl(false),
   });
 
+  ngOnInit(): void {
+    this.auth.appState$.subscribe((appState) => {
+      this.appState = appState.deeply.nested.value;
```

Comment:
I am confused with this example. The `appState` is the `appState`, and not some deeply nested value.
So I would either use `this.appState = appState` or `this.someOtherProperty = appState.deeply.nested.value`;

That said, even though it's good knowing this is possible, I would reduce the nesting here as I believe we want to avoid people using appState to track too much complexity. I would personally stick to a single property on the `appState`.